docs/pages/enterprise/workflow/ssh-approval-jira-server.mdx

```diff
@@ -179,22 +182,22 @@ The `[teleport]` section describes where is the teleport service running, and wh
 
 The `[jira]` section requires a few things:
 
-1. Your Jira Cloud or Jira Server URL. For Jira Cloud, it looks something like yourcompany.atlassian.net.
-2. Your username on Jira, i.e. benarent **Note: Not your email address.**
-3. Your Jira API token. **For Jira Server, this is a password.  it's a good idea to create a separate user record with permissions limited to accessing this particular project board, and use this with the bot.**
-4. And the Jira Project key, available in Project settings.
+1. Your Jira Cloud or Jira Server URL. For Jira Cloud, it looks something like `yourcompany.atlassian.net`.	
+2. Your username on Jira, i.e. `benarent` **Note: Not your email address.**	
+3. Your Jira API token. **For Jira Server, this is a password.  it's a good idea to create a separate user record with permissions limited to accessing this particular project board, and use this with the bot.**	
+4. And the Jira Project key, available in "Project settings".	
 
-`[http]` setting block describes how the Plugin's HTTP server works. The HTTP server is responsible for listening for updates from Jira, and processing updates, like when someone drags a task from Inbox to Approved column.
+`[http]` setting block describes how the Plugin's HTTP server works. The HTTP server is responsible for listening for updates from Jira, and processing updates, like when someone drags a task from the "Inbox" to the "Approved" column.
 
-You must provide an address the server should listen on, and a certificate to use, unless you plan on running with `--insecure-no-tls`, which we don't recommend in production.
+You must provide an address the server should listen on, and a certificate to use unless you plan on running with `--insecure-no-tls`, which we don't recommend in production.
 
 ## Testing
 
 You should be able to run the Teleport plugin now!
 
 ```bash
-teleport-jira start
-INFO   Starting Teleport Access JIRAbot 0.1.0-alpha.3:teleport-jira-v0.1.0-alpha.3-0-gea1ef8e jira/app.go:74
+sudo teleport-jira start
+# INFO   Starting Teleport Access JIRAbot 0.1.0-alpha.3:teleport-jira-v0.1.0-alpha.3-0-gea1ef8e jira/app.go:74
```

Comment:
FYI this example commented these lines, I followed suit and did the same in the other articles.